src/dhtproto/client/mixins/NeoSupport.d

```diff
@@ -290,6 +292,40 @@ template NeoSupport ( )
             return id;
         }
 
+        /***********************************************************************
+
+            Assigns a Remove request, removing a record from the specified
+            channel.
+            See $(LINK2 dhtproto/client/request/Remove.html, dhtproto.client.request.Remove)
+            for detailed documentation.
+
+            Params:
+                channel = name of the channel to remove a record from
+                key = hash of the record to remove
+                notifier = notifier delegate
+
+            Returns:
+                id of newly assigned request
+
+            Throws:
+                NoMoreRequests if the pool of active requests is full
+
+        ***********************************************************************/
+
+        public RequestId remove ( cstring channel, hash_t key,
+            Remove.Notifier notifier )
+        {
+            auto params = Const!(Internals.Remove.UserSpecifiedParams)(
+                Const!(Remove.Args)(channel, key),
+                Const!(Internals.Remove.UserSpecifiedParams.SerializedNotifier)(
+                    *(cast(Const!(ubyte[notifier.sizeof])*)&notifier)
```

Comment:
Has this been improved in a later version of swarm?

Reply:
Yes, it's part of swarm v5.0.0. This branch of dhtproto is currently on swarm v4.6.0. (I wanted to get all these PRs merged before merging dhtproto v14.x.x -> neo.)